addons/payment_mollie_official/static/src/js/payment_mollie_official.js

```diff
@@ -0,0 +1,398 @@
+odoo.define('payment_mollie_official.mollie_payment_form', function (require) {
+"use strict";
+

```

Comment:
I can see that the JS code has been very minimally migrated:
- still uses jQuery promises instead of native ones
- seem to support s2s in the JS but not in the python, so I assume it's a copy/paste file

not gonna review it now given the rest of my review up to now, there are bigger issues to address